doc/release/3.0.0.rst

```diff
@@ -0,0 +1,652 @@
+Tarantool 3.0
+=============
+
+Release date: December 26, 2023
+
+Releases on GitHub: :tarantool-release:`3.0.0`
+
+The 3.0 release of Tarantool introduces a new declarative approach for configuring a cluster,
+a visual tool for managing, configuring, and monitoring Tarantool EE clusters,
+and many other new features and fixes.
+This document provides an overview of the most important features for the Community and Enterprise editions.
+
+-   :ref:`New declarative configuration <3-0-new_declarative_configuration>`
+-   :ref:`Tarantool Cluster Manager <3-0-tarantool_cluster_manager>`
+-   :ref:`Administration and maintenance <3-0-enhancements_for_administration_and_maintenance>`
+-   :ref:`Developing applications <3-0-features_for_developers>`
+-   :ref:`Stability <3-0-stability>`
+
+
+
+.. _3-0-new_declarative_configuration:
+
+New declarative configuration
+-----------------------------
+
+Starting with the 3.0 version, Tarantool provides the ability to :ref:`configure the full topology of a cluster <configuration>` using a declarative YAML configuration instead of configuring each instance using a dedicated Lua script.
+Using a new approach, you can use local configuration in a YAML file for each instance or store configuration data in one reliable place, for example, a Tarantool or an etcd cluster.
+
+The example below shows how a configuration of a small sharded cluster might look.
+In the diagram below, the cluster includes 5 instances: one router and 4 storages, which constitute two replica sets.
+For each replica set, the master instance is specified manually.
+
+.. image:: _images/sample_cluster_diagram.png
+    :align: left
+    :width: 700
+    :alt: Cluster topology
+
+The example below demonstrates how a topology of such a cluster might look in a YAML configuration file:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/config.yaml
+    :start-at: groups:
+    :language: yaml
+    :dedent:
+
+You can find the full sample in the GitHub documentation repository: `sharded_cluster <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/sharding/instances.enabled/sharded_cluster>`_.
+
+The latest version of the tt utility provides the ability to :ref:`manage Tarantool instances <admin-start_stop_instance>` configured using a new approach.
+You can start all instances in a cluster by executing one command, check the status of instances, or stop them:
+
+.. code-block:: console
+
+    $ tt start sharded_cluster
+       • Starting an instance [sharded_cluster:storage-a-001]...
+       • Starting an instance [sharded_cluster:storage-a-002]...
+       • Starting an instance [sharded_cluster:storage-b-001]...
+       • Starting an instance [sharded_cluster:storage-b-002]...
+       • Starting an instance [sharded_cluster:router-a-001]...
+
```

Comment:
I find "Learn more in <>" sentences in the end of what's new sections more helpful than just links under some words in descriptions. Anyway this is minor, up to you.

Reply:
I'd keep this as is because `Learn more` phrases distract a user from scanning a what's new. But this definitely works better for regular topics.